agent/kibana/client.go

```diff
@@ -97,12 +97,6 @@ func NewConfigFromURL(kURL string, CAs []string) (*Config, error) {
 	c.Username = username
 	c.Password = password
 
-	if len(CAs) > 0 {
-		c.TLS = &tlscommon.Config{
-			CAs: CAs,
```

Comment:
do we need this option now? we use this to pass CAs as a root CAs to the client. but without having anything to pass i think we can remove it

Reply:
There are some case that we will still need to support passing a CAs, but this can done by altering the TLS field after.